spec/support/rails_template.rb

```diff
@@ -57,8 +65,14 @@
   ENV['BUNDLE_GEMFILE'] = "#{File.expand_path(ENV['BUNDLE_GEMFILE'])}"
 RUBY
 
+# Setup webpacker if necessary
+if webpacker_app
+  rake "webpacker:install"
+  gsub_file 'config/webpacker.yml', /^(.*)extract_css.*$/, '\1extract_css: true' if ENV['RAILS_ENV'] == 'test'
```

Comment:
Does it make sense to include this line inside the installer? We could replace only the `test:` section in the `webpacker.yml` file.

Reply:
I don’t think we should force that on client app. We only do so to test that AA assets are present

Reply:
Ok, yeah, if this is not the default we should change that. You're right :+1:.